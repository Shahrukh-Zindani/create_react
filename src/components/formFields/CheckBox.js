import React from 'react'


const CheckBox = ({ value, name, checked, onClick }) =>
  <input type="checkbox" name={name} value={value} checked={checked} onClick={onClick} />

export default CheckBox
