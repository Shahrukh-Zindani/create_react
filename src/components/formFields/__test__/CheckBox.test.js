import React from 'react'
import renderer from 'react-test-renderer'
import CheckBox from '../CheckBox'

describe('Input Field tests', () => {
  it('matches snapshot', () => {
    const tree = renderer.create(
      <CheckBox />
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
