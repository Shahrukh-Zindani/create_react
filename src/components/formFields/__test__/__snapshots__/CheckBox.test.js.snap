exports[`Input Field tests matches snapshot 1`] = `
<input
  checked={undefined}
  name={undefined}
  onClick={undefined}
  type="checkbox"
  value={undefined} />
`;
